graphbolt/src/cuda/utils.h

```diff
@@ -0,0 +1,36 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ *
+ * @file utils.h
+ * @brief CUDA utilities.
+ */
+
+#ifndef GRAPHBOLT_CUDA_UTILS_H_
+#define GRAPHBOLT_CUDA_UTILS_H_
+
+namespace graphbolt {
+namespace cuda {
+
+// The cache line size of GPU
+#define GPU_CACHE_LINE_SIZE 128
+// The max number of threads per block
+#define CUDA_MAX_NUM_THREADS 1024
+
+/** @brief Calculate the number of threads needed given the dimension length.
```

Comment:
What's "dimension length"? dimension or length? of length of the feature?

Reply:
Should be "the size of the dimension to be processed". In the context of IndexSelectSingleKernel, it is the size of `index`.